pages/common/jadx.md

```diff
@@ -0,0 +1,12 @@
+# jadx
+
+> Dex to Java decompiler.
```

Comment:
What about adding an extended description like this? (I personally do not know about "Dex" files but I guess what Android files means 😉)

```suggestion
> Dex to Java decompiler.
> Produces Java source code from Android Dex and Apk files.
```